packages/marginfi-v2-ui-state/src/lib/mrgnlend.ts

```diff
@@ -100,10 +100,14 @@ function makeBankInfo(bank: Bank, oraclePrice: OraclePrice, emissionTokenData?:
 
   if ((bank.emissionsActiveLending || bank.emissionsActiveBorrowing) && emissionTokenData) {
     const emissionsRateAmount = new BigNumber(nativeToUi(bank.emissionsRate, emissionTokenData.decimals));
-    const emissionsRateValue = emissionsRateAmount.times(emissionTokenData.price);
-    const emissionsRateAdditionalyApy = emissionsRateValue.div(getPriceWithConfidence(oraclePrice, false).price);
 
-    emissionsRate = emissionsRateAdditionalyApy.toNumber();
+    // why are we multiplying emissionsRateAmount by emissionTokenData.price and then dividing by oracle price?
```

Comment:
@k0beLeenders

Reply:
I def looks intentional that were multiplying and dividing it by the price but can't figure out why we did it